warehouse/upload.go

```diff
@@ -716,7 +704,7 @@ func (job *UploadJobT) loadUserTables() (loadErrors []error, tableUploadErr erro
 		return []error{err}, nil
 	}
 
-	if !loadTables {
+	if !hasLoadFiles {
```

Comment:
In the case where prev upload has {identifies: S, users: F}, should we load identifies in the current upload? `hasLoadFiles` will always be true for current upload right because we do not skip load file generation for selective tables ?

Reply:
In the case you mentioned, we would be breaking with an error with this check -> `if prevJobID, ok := previouslyFailedTables[tName]; ok {`

So even before this `if !hasLoadFilesCheck {` is done, we would be returning from the function with non-nil error.

I had left the hasLoadFilesCheck just because it was already being done. We can remove it if we are confident that there is no way we arrive here without loadFiles. Infact we can remove such checks for other table exporting steps as well?